src/styles/index.scss

```
@@ -1,3 +1,116 @@
+
+/* import of the review stars block */
+@import './stars';
+
+/* import of the review stars block */
+
+/* start the variables of repeated colors */
+$button-blue: #00acdc;
+$color-dark-blue: #060b35;
+$color-dark-grey: #616070;
+$white-color: #fff;
+
+/*end the variables of repeated colors*/
+
+/* #start of the main page styles */
+* {
+  font-size: 14px;
+  font-weight: 700;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  font-weight: 400;
+}
+
+h2,
+p {
   margin: 0;
+  font-weight: 400;
+}
+/* #end of the main page styles */
+
+/* #start of  top and middle the card styles */
+.cards {
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  width: 200px;
+  display: flex;
+
+  &_product-image {
```

use double `_` to name an element part

```suggestion
  &__product-image {
```

---

fixed